openwisp_utils/admin_theme/dashboard.py

```diff
@@ -21,6 +21,17 @@ def _validate_chart_config(config):
     assert not ('group_by' in query_params and 'annotate' in query_params)
     if 'annotate' in query_params:
         assert 'filters' in config, 'filters must be defined when using annotate'
+    if 'quick_link' in query_params:
+        assert (
+            'url' in query_params['quick_link']
+        ), 'url must be defined when using quick_link'
+        assert (
+            'label' in query_params['quick_link']
+        ), 'label must be defined when using quick_link'
+        if 'custom_css_classes' in query_params['quick_link']:
+            assert isinstance('custom_css_classes', list) or isinstance(
+                'custom_css_classes', tuple
+            ), 'custom_css_classes must be either a list or a tuple'
```

Comment:
please add some assertions in `test_register_dashboard_template` for these new instructions.